### PR TITLE
Ui-integration of staking page wrt design

### PR DESCRIPTION
**Pull request by Sanjeev1308**
To test:
1. Test the design for staking page as per figma https://www.figma.com/file/aYVD9iZnfmsEKzlN2I3dtg/Axial-V2-UI-Designs?node-id=3288%3A621734
2. Check the staking list item showing correct info(sAxial and veAxial)
3. Check the deposit flow on the sAxial and veAxial
4. Check the withdraw flow on the sAxial and veAxial

Need to work:
Responsiveness for mobile

### CODE REVIEW

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-staking-design.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-staking-design.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-staking-design.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-staking-design.dev-app.axial.exchange

File: src/app/components/base/snowDatePicker/index.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {
 } from "@mui/x-date-pickers";
 import { FC } from "react";
 import { AdapterDateFns } from "@mui/x-date-pickers/AdapterDateFns";
-import { BaseInputStyles } from "styles/baseInputStyles";
+// import { BaseInputStyles } from "styles/baseInputStyles";
```
Comment (saito-sv) [inline src/app/components/base/snowDatePicker/index.tsx]: move unused code.

File: src/app/components/base/snowDatePicker/index.tsx [hunk 2]
```diff
@@ -31,7 +31,9 @@ export const SnowDatePicker: FC<Props> = ({ value, onChange, ...rest }) => {
 };
 const StyledTextField = styled(TextField)(({ theme }) => ({
   "& .MuiInputBase-input": {
-    ...BaseInputStyles({ theme }),
+    // ...BaseInputStyles({ theme }),
```
Comment (saito-sv) [inline src/app/components/base/snowDatePicker/index.tsx]: move unused code.

File: src/app/components/base/snowDatePicker/index.tsx (hunk 1 above)
Comment (Sanjeev1308) [reply src/app/components/base/snowDatePicker/index.tsx]: Done

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-staking-design.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-staking-design.dev-app.axial.exchange

**Review by saito-sv — APPROVED** [review]

**Pull request merged**